__tests__/app.test.js

```diff
@@ -11,4 +11,16 @@ describe('zodiac routes', () => {
     });
     expect(res.body).toEqual(expected);
   });
-});
+
+  it('/zodiac/:id should return zodiac detail', async () => {
```

Comment:
the rubric is asking for routes built vertically. You want to make sure that you are testing first, then building the route one at a time.